stopcovid/sms/send_sms.py

```diff
@@ -50,6 +50,8 @@ def _send_batch(batch: SMSBatch) -> Optional[List[MessageInstance]]:
         return None
     twilio_responses = []
     for i, message in enumerate(batch.messages):
+        if (message.body is None) and (message.media_url is None):
+            continue
```

Comment:
What are your thoughts on raising an exception here instead?

Reply:
that's already effectively what's happening

Reply:
when the twilio API raises an error

Reply:
ohh true

Reply:
i'm open to that--my thinking in this case was that this can realistically happen with custom courses, and rather than try to validate more precisely in the dashboard, we can just take a more lenient here 🤷‍♀️  but I could also see how this might be something we'd want to know about if we create a bug that accidentally removes text/media from messages that should have them or something

Reply:
yeah my first thought was it's a sign something is wrong if we try to send an empty message but idk I can see it both ways 👍